meta-flutter/README.md

```diff
@@ -39,5 +77,20 @@ You need to install libsystemd in the same way with the DRM-GBM backend.
 $ bitbake flutter-drm-eglstream-backend
 ```
 
+## Cross-building with Yocto SDK
+Installation of the cross-toolchain creates a cross-toolchain environment setup script in the directory that Yocto SDK was installed. Source the cross-toolchain environment setup file :
+```Shell
+$ source /opt/poky/3.1.7/environment-setup-aarch64-poky-linux
+```
+
+Setting environment variables :
+```Shell
+$ export CC=${CLANGCC}
+$ export CXX=${CLANGCXX}
+```
+
+Build is the same way as self-build, see: [3.1. Self-build](../doc/README.md)
```

Comment:
After doing that, you can build the embedder as normal like self-building on hosts. It means you don't need to be aware of cross-building. See: [3.1. Self-build](../doc/README.md)